<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passwd">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Ensure your password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_surepasswd">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Sure">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3399ff;       /* 主体背景色 */
    color: white;                    /* 字体颜色 */
    border-radius: 10px;             /* 圆角半径 */
    padding: 5px 10px;               /* 内边距 */
    font-size: 14px;
    font-weight: bold;
    border: 1px solid #007acc;       /* 边框颜色 */
}

QPushButton:hover {
    background-color: #007acc;       /* 悬浮时背景色 */
}

QPushButton:pressed {
    background-color: #005f99;       /* 按下时背景色 */
}
</string>
   </property>
   <property name="text">
    <string>Sure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3399ff;       /* 主体背景色 */
    color: white;                    /* 字体颜色 */
    border-radius: 10px;             /* 圆角半径 */
    padding: 5px 10px;               /* 内边距 */
    font-size: 14px;
    font-weight: bold;
    border: 1px solid #007acc;       /* 边框颜色 */
}

QPushButton:hover {
    background-color: #007acc;       /* 悬浮时背景色 */
}

QPushButton:pressed {
    background-color: #005f99;       /* 按下时背景色 */
}
</string>
   </property>
   <property name="text">
    <string>Click here to return login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Creat </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Account!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
